python-flask/app/simulator/routes.py

```diff
@@ -0,0 +1,125 @@
+from flask import Blueprint
+from flask import jsonify, request, g, abort
+from werkzeug.security import generate_password_hash
+from datetime import datetime
+from app.models.user import User, Follower
+from app.models.message import Message
+from app.models.latest import Latest
+from app.utils import not_req_from_simulator, update_latest, get_user_id
+from app.extensions import db
+
+sim_bp = Blueprint('sim', __name__)
```

Comment:
I would change the sim_bp and 'sim' to api to make it clear

Reply:
Changed everything which was sim or simulator to api.